core/utils/poly-look/src/react-components/chipGroup/chipGroup.css

```diff
@@ -2,14 +2,14 @@
   height: 32px;
   margin: 4px;
   padding: 0 12px;
-  border: solid 1px var(--theme-primary-color);
+  border: solid 1px var(--theme-secondary-color);
```

Comment:
Er. This is confusing, and the change even more. If it's not a fallback color, maybe call them "light" and "dark".

Reply:
Can't in the lexicon feature for example primary would be white and secondary light red

Reply:
I decided the background is the primary color: light theme -> white background -> chipsborder needs to be blue (I just got it wrong prior to this)

Reply:
Why not simply background and foreground then? No decision on names is needed then.

Reply:
But thats not the case it's not just background and foreground, see chipsgroup activated. It's just a color palette which is used. I don't want to bind it to certain elements like background

Reply:
the problem is that, at the end of the day, what's primary and secondary is a judgment call, which can change at any moment (and indeed, it has). Choosing a more semantic, function-based denomination might possibly need less thinking. But I can't really think of one right now, so... I guess it's OK.

Reply:
I think it should not be a judgement call but something we develop with the designer meaning: "In this App we use a light theme with blue elements in it and a dark theme with ..." And then we can make the background always the primary color and the contrasting one the secondary (and tertiary)